src/notify.rs

```diff
@@ -85,6 +97,9 @@ impl Notifier {
 
     /// Invalidate the kernel cache for a given directory entry and inform
     /// inotify watchers of a file deletion.
+    /// # Errors
+    /// Returns an error if the notification data is too large.
+    /// Propagates errors due to communicating with the fuse device.
```

Comment:
`Propagates errors due to communicating with the fuse device.`

Let's remove this in all the messages. I don't think it adds useful information. Returning an `io::Result` already explains that it propagates errors

Reply:
I guess that is a fair criticism.